zerver/migrations/0424_backfill_remote_realmauditlog_extradata_to_json_field.py

```diff
@@ -0,0 +1,72 @@
+# Generated by Django 4.0.7 on 2022-09-30 20:30
+
+import ast
+from typing import Type
+
+from django.db import migrations
+from django.db.backends.postgresql.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from django.db.models import JSONField, Model
+from django.db.models.functions import Cast, JSONObject
+
+USER_FULL_NAME_CHANGED = 124
+BATCH_SIZE = 5000
+
+
+def do_bulk_backfill_extra_data(
+    audit_log_model: Type[Model], id_lower_bound: int, id_upper_bound: int
+) -> None:
+    # First handle the special case for audit logs with the USER_FULL_NAME_CHANGED event, which stores
+    # the full name not as str(dict()) but a plain str.
+    audit_log_model.objects.filter(
+        event_type=USER_FULL_NAME_CHANGED, id__range=(id_lower_bound, id_upper_bound)
+    ).update(extra_data_json=JSONObject(old_name="extra_data"))
+
+    # A dict converted with str() will start with a open bracket followed by a single quote.
```

Comment:
To be slightly more explicit (and make sure I understand!):
```suggestion
    # A dict converted with str() will start with a open bracket followed by a single quote,
    # as opposed to a JSON-encoded value, which will use a _double_ quote.